demos/videohelpdesk/README.md

```diff
@@ -102,3 +102,7 @@ see both local and remote videos in their Help Desk Widget.
 
 Press the **Join Next** button again to go to the next call, and so on until the queue
 is drained.
+
+### Cleaning up
+To avoid incurring future charges, please delete any resources in your account that you are not using. 
+You can delete the CloudFormation stack, which will clean up all associated resources
```

Comment:
Suggest: To avoid incurring any unintended charges as a result of deploying the video help desk, it is important to delete the AWS Cloud9 environment and AWS CloudFormation stack after you are finished using them.

Reply:
Updated.